add sedai (#1096)
DataDog/integrations-extras: sedai/README.md

> # [Sedai][1] — Find out why production is the new playground with a free 14 day trial

Reviewer: I don't believe level 1 headings are supported either - I'd convert this to a level 3 or just leave it as body text.

Author: done

> ## Continuous Autonomous Availability Management (CAAM)

Author: done

> ## Continuous Autonomous Availability Management (CAAM)

Reviewer: We'll have to remove this level 2 heading as well

Author: done

*Intelligent enough to detect, independent enough to act — that's the power of autonomous*

> ## Overview

Reviewer: Under the Overview tab, I'd make it clear that Sedai is offering an integration with Datadog, and that if you're already a customer of Sedai you can set up the integration quickly by following the instructions in the Configuration tab.

Author: done

**Unlock the full potential of your microservices:**
* Achieve 99.99% or better application availability
* Autonomously reduce resource latency by 75%
* Reduce cloud costs by 30% through autonomous optimization
* Increase team productivity and agility by 3x

## Core Features
* **Easy setup:** Our agentless approach integrates with your cloud accounts and observability platforms to get up and running in minutes.
* **Configuration-free:** We infer your topology and metrics, and intelligently leverage them to detect signals based on observed performance.
* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
* **Release Intelligence:** Validate releases' performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.

> ## Home Page

Author: changed to "Visit **[Sedai Inc.][1]** to learn more.**"

* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that smartly manages your resources based on their traffic needs.

> Visit **[Sedai Inc.][1]** to learn more.**

Reviewer: ```suggestion
Visit **[Sedai Inc.][1]** to learn more.
```

Author: done

> ### Technical Documentation
> Please review the [Sedai User Guide][4].

Reviewer 2: We include a link to the user guide and the documentation already below.

```suggestion
```

## Setup

> Existing Sedai customers can **[learn how to receive notifications in DataDog about Sedai activity.][6]**

Reviewer: ```suggestion
Existing Sedai customers can receive notifications in DataDog about Sedai activity. Learn how to connect Datadog to Sedai in **[our documentation][6]**
```

Author: done

> Existing Sedai customers can receive notifications in DataDog about Sedai activity. Learn how to connect DataDog to Sedai in **[our documentation][6].**

Reviewer 2: [6] links to https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications, not https://sedai.gitbook.io/sedai/sedai-user-guide/controls/integrations/notifications-integrations. There are two sections titled "Notifications" so I want to confirm we're linking to the right one!  

```suggestion

After connecting your cloud provider, monitoring provider, and notification provider accounts, Sendai identifies your topology and metrics and starts to analyze performance.

### Installation

Existing Sedai customers can receive notifications about Sedai activity in Datadog. For more information, see the [Sedai documentation][6].
```

Author: The two links are on purpose, our UI has 2 related settings pages that are involved in configuration of notifications.

**[Schedule a free 30 minute demo to learn more][2]**

**[Schedule a free 14-day trial][2]**

[Contact us][5] if you have questions about the process.

> For existing Sedai customers, **[learn how to receive notifications in DataDog about Sedai activity.][6]**

Author: Sedai customers who are also DataDog customers will already have an integration set up on Sedai's side, the instructions tell them how to configure specific notifications about Sedai activity to appear on their DataDog Dashboard.

> ### Ready to Autonomously Optimize your applications for heightened performance?
> With our agentless approach, integration is seamless and setup takes minutes — simply connect your cloud and monitoring accounts and favorite integrations. We automatically identify your topology and metrics and start analyzing performance immediately.

Reviewer 2: ```suggestion
```

> **[Schedule a free 30 minute demo OR a free 14-day trial][2]**

Reviewer 2: ```suggestion
## Data Collected

### Metrics

This integration does not collect metrics.

### Events

This integration does not support events.

### Service

This integration does not collect service checks.

## Support

Need help? Email [Support][3] is available 24/7.

Alternatively, [schedule a 30-minute demo][2] or [contact us][5] for a [free 14-day trial][2]. 
```

> [Contact us][5] if you have questions about the process.
> 
> ## Support
> Support is provided 24/7 via email to [support@sedai.io][3].

Reviewer 2: ```suggestion
```

> [1]:[https://www.sedai.io]
> [2]:[https://calendly.com/sedai/sedai-demo]
> [3]:[email:support@sedai.io]
> [4]:[https://sedai.gitbook.io/sedai/]
> [5]:[email:contact@sedai.io]
> [6]:[https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications]

Reviewer 2: ```suggestion
[1]: https://www.sedai.io
[2]: https://calendly.com/sedai/sedai-demo
[3]: mailto:support@sedai.io
[4]: https://sedai.gitbook.io/sedai/
[5]: mailto:contact@sedai.io
[6]: https://sedai.gitbook.io/sedai/sedai-user-guide/controls/notifications
```

# Autonomous Cloud Application Management

*Autonomous is your future; Production is the new playground*

> ## **[Schedule a Demo][2]**

Reviewer 3: ```suggestion
```
Let's have this section in one place only

Author: done

* Achieve 99.99% or better application availability
* Autonomously reduce resource latency by 75%
* Reduce cloud costs by 30% through autonomous optimization
* Increase team productivity and agility by 3x

### Core Features

* **Easy setup:** Sedai's agentless approach integrates with your cloud accounts and observability platforms so you can get up and running in minutes.
* **Configuration-free:** By inferring your topology and metrics, Sedai leverages them to detect signals based on observed performance.
* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
* **Release Intelligence:** Validate release performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that manages your resources based on traffic needs.
* **Smart SLOs:** In addition to automatically analyzing and defining targets, Sedai manages error budgets with autonomous actions that prevent errors, downtime, and SLO breaches.

> To learn more, see [Sedai Inc.][1].

Reviewer: ```suggestion
To learn more, see [Sedai Inc.][1]
```

## Setup

> After connecting your cloud provider, monitoring provider, and notification provider accounts, Sendai identifies your topology and metrics and starts to analyze performance.

Reviewer: Doing a final read-through and found this typo: 
```suggestion
After connecting your cloud provider, monitoring provider, and notification provider accounts, Sedai identifies your topology and metrics and starts to analyze performance.
```

Software as a Service (SaaS)

## Pricing
### Sedai Autonomous Cloud Application Management
| Tier 	              | Features               | app instances | Lambda invocations | 1 MONTH | 12 MONTHS |
| :----	              | :------:               | :-----------: | :----------------: | :-----: | :-------: |
| Starter	          | Core Features          |       100     |        20M 	    |    $90  |    $900   |
| Professional Tier 1 | Starter + Remediations |       100     |        20M 	    |   $340  |  $3,420   |
| Professional Tier 2 | Starter + Remediations |       500     |       100M 	    | $1,300  | $13,000   |
| Professional Tier 3 | Starter + Remediations |     2,000     |       500M 	    | $4,800  | $48,000   |

### Additional usage fees
You will be billed monthly for additional usage costs if your usage exceeds your contract. Your additional usage costs will be determined by the number of units you use above your contract.

| Tier       	      | per instance/pod | per 10k invocations |
| :---       	      | :--------------: | :-----------------: |
| Professional Tier 1 |      $3.40       |       $0.170        |
| Professional Tier 2 |      $2.60       |       $0.130        |

> | Professional Tier 3 |      $2.40       |       $0.096        |

Reviewer 3: Could this be removed from the readme, and instead have a link to sedai website having the latest information?

Author: done
